### PR TITLE
Clang format 2

### PR DESCRIPTION
**Pull request by rsetaluri**

### CODE REVIEW

**Comment by rsetaluri** [comment]
@rdaly525 compare to #861

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+Language:        Cpp
+AccessModifierOffset: -1
```
Comment (rsetaluri) [inline .clang-format:3]: like this as -1 vs -2

**Comment by leonardt** [comment]
LGTM, is there a simple way to integrate this into CI (e.g. block if clang-format hasn't been run).  Perhaps we can run clang-format and use git to see if there's any diffs, and set an error code if so

**Comment by rsetaluri** [comment]
Solved by #879

**Pull request closed without merging**